scanner/decisions/0014-csaf-enricher.md

```diff
@@ -0,0 +1,94 @@
+# 0014 - CSAF Enricher
```

Comment:
I may need to change this if https://github.com/stackrox/stackrox/pull/13752 is merged first

Reply:
I think this should go first; It sets the right expectation regarding the possibility of scores (EPSS and CVSS) flips.

Reply:
Looks like the EPSS ADR went in first, but it has caveats about the work on this PR.

Reply:
yep, I'll adjust as-needed